lib/fluent/plugin/out_splunk_hec.rb

```diff
@@ -239,7 +239,7 @@ def pick_custom_format_method
     def format_event(tag, time, record)
       MultiJson.dump({
 	host: @host ? @host.(tag, record) : @default_host,
-	time: time.to_i
+	time: time.to_f
```

Comment:
According to the reference, time should be a string if it's not an integer: http://docs.splunk.com/Documentation/Splunk/latest/RESTREF/RESTinput#services.2Fcollector
Although it still works using a float number, but we should comply with the reference.